File: fsharp-giraffe-server/traveltimeplatform/src/model/ResponseGeocoding.fs
namespace traveltimeplatform.Model

open System
open System.Collections.Generic
open traveltimeplatform.Model.ResponseGeocodingGeoJsonFeature

module ResponseGeocoding = 

  //#region ResponseGeocoding


  type ResponseGeocoding = {
    Type : string;
    Features : ResponseGeocodingGeoJsonFeature[];
  }
  //#endregion
  

File: scala-finch/src/main/scala/org/openapitools/models/ResponseGeocoding.scala
package org.openapitools.models

import io.circe._
import io.finch.circe._
import io.circe.generic.semiauto._
import io.circe.java8.time._
import org.openapitools._
import org.openapitools.models.ResponseGeocodingGeoJsonFeature
import scala.collection.immutable.Seq

/**
 * 
 * @param _type 
 * @param features 
 */
case class ResponseGeocoding(_type: String,
                features: Seq[ResponseGeocodingGeoJsonFeature]
                )

object ResponseGeocoding {
    /**
     * Creates the codec for converting ResponseGeocoding from and to JSON.
     */
    implicit val decoder: Decoder[ResponseGeocoding] = deriveDecoder
    implicit val encoder: ObjectEncoder[ResponseGeocoding] = deriveEncoder
}
